recipes/newrelic/infrastructure/ubuntu.yml

```diff
@@ -261,7 +261,7 @@ install:
                   # Process-based validation attempt
                   if infra_present; then
                     echo "detected newrelic-infra process running"
-                    break
+                    exit 0
```

Comment:
It seems, the `break` statement is not breaking out of the while loop. I do not like using an `exit 0` but it works fine in my tests (and Pavan's). Please: Is using `exit 0` here acceptable in your opinion @Julien4218, @NRhzhao?

![image](https://user-images.githubusercontent.com/102765645/194888190-940c0cb8-dfbf-468e-8658-35fbade2ab0f.png)

Reply:
@amolero-nr seems fine to me, since we've validated infra, I don't see what else is needed to be done in the recipe?
If you do, make sure you make the change in all the recipes.

Reply:
Thank you!  Will update the rest of them with this fix.